### PR TITLE
Added warning about parser limitations

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: _includes/manuals/1.13/4.3.4.2_isc_dhcp.md [hunk 1]
```diff
@@ -75,3 +74,8 @@ subnet 10.1.1.0 netmask 255.255.255.0 {
 
 }
 </pre>
+
+At the moment, foreman proxy does not recognize all configuration syntax and
```
Comment (domcleal) [inline _includes/manuals/1.13/4.3.4.2_isc_dhcp.md]: "foreman proxy" => "Smart Proxy" as used in this doc

File: _includes/manuals/1.13/4.3.4.2_isc_dhcp.md [hunk 2]
```diff
@@ -75,3 +74,8 @@ subnet 10.1.1.0 netmask 255.255.255.0 {
...
+parsing is limited to one or more subnet blocks with routers,
+domain-name-servers and range options. These must not be enclosed in pool
```
Comment (domcleal) [inline _includes/manuals/1.13/4.3.4.2_isc_dhcp.md]: Use backticks around routers, d-n-s, range, and pool to indicate they're config directives.

File: _includes/manuals/1.13/4.3.4.2_isc_dhcp.md [hunk 3]
```diff
@@ -75,3 +74,8 @@ subnet 10.1.1.0 netmask 255.255.255.0 {
...
+blocks as these are ignored by proxy's config parser.
```
Comment (domcleal) [inline _includes/manuals/1.13/4.3.4.2_isc_dhcp.md]: "the proxy's" or "the Smart Proxy's"

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by lzap** [comment]
All done, added tested workaround.

**Comment by domcleal** [comment]
Is this only applicable to 1.13? Is it fixed by #16079? If not, please add to the 1.14 docs too.

**Comment by lzap** [comment]
Applicable to both, added.

**Comment by domcleal** [comment]
Sorry, I've just re-read the commit and looked at the test (https://github.com/theforeman/smart-proxy/pull/474/files#diff-0c16be9e3188140992c2d2092eda3f3d), shouldn't that have fixed it? In which case this is only applicable to 1.13.

**Comment by lzap** [comment]
Yeah you're right, rebased.

**Review by domcleal — APPROVED** [review]

**Comment by domcleal** [comment]
Merged, thanks @lzap.

**Pull request merged**